aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -294,6 +295,14 @@ public class AwsProperties implements Serializable {
 
   public static final String DYNAMODB_TABLE_NAME_DEFAULT = "iceberg";
 
+  public static final String DYNAMODB_V2_SCHEMA_DEFAULT_TABLE_NAME = "iceberg_v2";
+
+  public static final String DYNAMODB_CATALOG_SCHEMA_FORMAT =
```

Comment:
I think we can use a simpler name, just `dynamodb.schema-version` should work? Any reason to create sublevels?

Reply:
Idea was to indicate that this property is related to DynamoDb catalog, essentially separate out/differentiate from DynamoDb lock manager.
Let me know If I should follow current naming convention. Also, what about `dynamodb.catalog.schema-version` instead of existing `dynamodb.catalog.schema.format-version`

Reply:
modified the config property to `dynamodb.catalog.schema-version`. let me know if this looks good